Swamp Sneak/Assets/Scripts/Classes/Player.cs

```diff
@@ -2,7 +2,64 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+
 public class Player : Person {
+	private GameControl control;
+	private Stats player_stats;
+
+	/* From documentation: 
+	 * "Each GameObject's Awake is called in a random order between objects. Because of this, 
+	 * you should use Awake to set up references between scripts, and use Start to pass any 
+	 * information back and forth. Awake is always called before any Start functions."
+	 * 
+	 * "Use Awake instead of the constructor for initialization, as the serialized 
+	 * state of the component is undefined at construction time. Awake is called once, just like 
+	 * the constructor. 
+	 * Link: https://docs.unity3d.com/ScriptReference/MonoBehaviour.Awake.html
+	*/
+	void Awake() {
+		// Get a reference to the GameControl object
+		control = GameObject.FindGameObjectWithTag("GameController").GetComponent<GameControl>();
+	}
+
+	void Start() {
+		// If the save file does not exist, initialize player_stats to default values.
+		// If it does exist, the save file will be loaded and stored to a PersistentData
+		// object in GameControl's Awake function.
+		if (control.persistent_save_data == null) {
+			player_stats = new Stats ();
+			player_stats.level = 1;
+			player_stats.experience = 0;
+			player_stats.stealth = 1;
+
+		}
+		// Otherwise, use the persistent save data
+		else {
+			// Initialize player_stats
+			player_stats = new Stats ();
+			player_stats.level = control.persistent_save_data.player_level;
+			player_stats.experience = control.persistent_save_data.player_experience;
+			player_stats.stealth = control.persistent_save_data.player_stealth;
+		}
+	}
+
+	public int getStealth() {
+		return player_stats.stealth;
+	}
+
+	public int getLevel() {
+		return player_stats.level;
+	}
+
+	public int getExperience() {
+		return player_stats.experience;
+	}
 
+	// Container class to hold player stats
+	public class Stats {
```

Comment:
Not sure how I feel about having a subclass container containing variables to be used solely within the parent class. Ideally, these would just be private variables with public getters/setters (Which C# has special, easy, syntax for and can easily be created in VS by just typing `prop` then hitting Tab).

But, I'll allow it. 
Though, it should probably be private since there doesn't seem to be any other class that is using this "container".

Reply:
This was mostly for organizational purposes. We can get rid of it though, I'm fine either way!

Reply:
I feel you, it doesn't make much of a difference really :)